python_builtin_functions.py

```diff
@@ -4,86 +4,170 @@ def myabs(number):
         return -number
     return number
 
+
 def myall(iterable):
     """ Implementation of built-in `all` function """
-    pass
+    for item in iterable:
+        if not item:
+            return False
+    return True
 
 
 def myany(iterable):
     """ Implementation of built-in `any` function """
-    pass
+    for item in iterable:
+        if item:
+            return True
+    return False
 
 
 def mylen(s):
     """ Implementation of built-in `len` function """
-    pass
+    i=0
+    for _ in s:
+        i+=1
+    return i
 
 
 def mysum(iterable):
     """ Implementation of built-in `sum` function """
-    pass
+    i=0
+    for x in iterable:
+        i+=x
+    return i
 
 
 def mymax(iterable):
     """ Implementation of built-in `max` function """
-    pass
+    i=0
+    for x in iterable:
+        if x>i:
+            i=x
+        else:
+            continue
+    return i
 
 
 def mymin(iterable):
     """ Implementation of built-in `min` function """
-    pass
+    i=1000**1000
+    for x in iterable:
+        if x<i:
+            i=x
+        else:
+            continue
+    return i
 
 
 def mydivmod(a, b):
     """ Implementation of built-in `divmod` function """
-    pass
+    q=a//b
```

Comment:
Further refactoring ...

You can use a one liner as below..

`return (a//b, a%b)`